### DISCUSSION TITLE
Can not open 127.0.0.1:8000/docs   500 Internal Server Error

### DISCUSSION THREAD

**Discussion by LiNoZues** [fastapi/fastapi#8792] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
@router.post('/upload_case')
async def upload_case(case: UploadFile = File(...), auto_user: str = Header(...),
                      upload_type: str = Form(...),
                      csv: str = Form(None)):
```

### Description

if use File , The 127.0.0.1:xxxx/docs is not working

### Operating System

Windows

### Operating System Details

Error Prompt :   KeyError: <class 'pydantic.main.Body_upload_case_api_upload_case_post'>

### FastAPI Version

0.68.0

### Python Version

py 3.8

### Additional Context

![image](https://user-images.githubusercontent.com/46307844/130001495-637badae-0863-44f4-8a9c-20d1faa2524c.png)

**Comment by ghost** [comment, marked as answer]
I just replicated this scenario and it works* for me. If you don't mind can you share GIF that records what your trying to do and where exactly it breaks; start-to-end^? 🙂

*using py3.9, *nix OS and non RC version of any apps.
^this is to cross check diff of what steps I followed and your following are same

**Reply by tiangolo** [reply]
Thanks @prnam!

@LiNoZues if you're still having problems, please update the example with a self-contained example that I can copy-paste to reproduce your problem.

**Comment by hrittikhere** [comment]
It works for me too. As @prnam says a GIF would be awesome

**Comment by LiNoZues** [comment]
@hrittikhere  @prnam   I made a gif  but it's a big file , i can't upload it. In issues #3756 , i posted my code,and in pull requests# #3762 i fixed it, it works  but i dont konw if it's right

**Comment by g-roblin** [comment]
I have the same issue @LiNoZues .  I will try your fix.
